tests/support/test.tcl

```diff
@@ -141,6 +141,31 @@ proc wait_for_condition {maxtries delay e _else_ elsescript} {
     }
 }
 
+proc verify_replica_online {master replica_idx max_retry} {
+    while {$max_retry} {
+        set info [$master info]
+        set pattern *slave$replica_idx:*state=online*
+        if {[string match $pattern $info]} {
+            break
+        } else {
+            incr max_retry -1
+            after 100
+        }
+    }
+    if {$max_retry == 0} {
+        error "assertion:Replica not correctly synchronized"
+    } 
+}
+
+proc wait_for_value_to_propegate_to_replica {master replica key} {
+    set val [$master get key]
+    wait_for_condition 50 1000 {
```

Comment:
why not just use wait command?

Reply:
Yes, used in replicaiton.tcl for some tests there (like `Test rdb-channel psync established after rdb load, master keep repl-block`). 
We need this functionality in order to make server stop while continue running the test

Reply:
Wait command won't help here. The key that I wait on `wait_for_value_to_propegate_to_replica ` in some tests was created/changed by a background handler (using separate client). Wait command can only ensure that the current client commands were propagated.